guides/troubleshooting/docker-problems.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Docker problems
+description: Learn how to resolve issues related to Docker inside Coder workspaces.
+---
+
+The kernel allocates a system key for each container created. When lots of developers
+are sharing the same instance, limits can be hit.
+
+> docker: Error response from daemon: OCI runtime create failed:
+container_linux.go:370: starting container process caused:
+process_linux.go:459: container init caused: join session keyring:
+create session key: disk quota exceeded: unknown.
+
+You can use a DaemonSet to apply the following values to increase the limits:
+
+```
+sudo sysctl -w kernel.keys.maxkeys=20000
```

Comment:
suggest we expand on this and add a daemonset example, we can reuse some of the content from the inotify watch limit doc

Reply:
@jawnsy Do you mind reviewing the content I added re: DaemonSets? I'm not super familiar w/ these so am open to all suggestions